spec/requests/api/orchestration_template_spec.rb

```diff
@@ -125,4 +125,62 @@
       expect(OrchestrationTemplate.exists?(hot.id)).to be_falsey
     end
   end
+
+  context 'orchestration template copy' do
+    let(:test_ot_1) { FactoryGirl.create(:orchestration_template_cfn) }
+    let(:test_content_1) { "{ 'Description': 'Test content 1' }\n" }
+
+    it 'copy template with the same content' do
+      api_basic_authorize collection_action_identifier(:orchestration_templates, :copy)
+
+      run_post(orchestration_templates_url(test_ot_1.id), gen_request(:copy))
+      expect(response).to have_http_status(:bad_request)
+    end
+
+    it 'copy single template with the different content' do
+      api_basic_authorize collection_action_identifier(:orchestration_templates, :copy)
+
+      expected = {
+        'content' => test_content_1
+      }
+
+      expect do
+        run_post(
+          orchestration_templates_url(test_ot_1.id),
+          gen_request(:copy, :content => test_content_1)
```

Comment:
I believe it will save you from entering a new name, description and type. Also, we have the possibility to copy an orchestration template via UI, so it better to have such possibility via API.
I believe @chriskacerguis can explain the intention after this feature in the context of self-service.